### PR TITLE
complete dishes_control_api

### PR DESCRIPTION
**Pull request by Brianlih**
complete dishes_control_api spec

### CODE REVIEW

File: client/api_spec/dishes_control_api.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+## Admin Dishes_Control [/api/dishes_control]
+
+### Create Order to complete order list [POST /api/dishes_control/complete_order_list]
+
++ Request (application/json)
+    + Attributes
+        + dishes_id: 02543 (number, required) - write dishes_id to database.
+        + vendorname: `蘭媽` (string, required) -  write vendorname to database.
+        + dishes_name: `無骨雞排` (strimg, required) - write dishes_name to database.
+        + price: 80 (number, required) - write price to database.
```
Comment (barrychung1112) [inline client/api_spec/dishes_control_api.md:16]: This description should be tell others the meaning of those attributes.

Comment (Brianlih) [reply client/api_spec/dishes_control_api.md]: I have fixed it.

File: client/api_spec/dishes_control_api.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
++ Response 204 (application/json)
+    + Attributes
+        + error_message: `writed fail` (string, required) - return `writed fail` if writed fail.
+
++ Response 200 (application/json)
+    + Attributes
+        + message: `writed successful` (string, required) - return `writed successful` if writed successful.
```
[the remark this answers is not in the document]
Comment (Brianlih) [reply client/api_spec/dishes_control_api.md]: I have fixed it.

File: client/api_spec/dishes_control_api.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# Group Dishes_Control API
+
+## Admin Dishes_Control [/api/dishes_control]
+
+### Create Order to complete order list [POST /api/dishes_control/complete_order_list]
+
++ Request (application/json)
+    + Attributes
+        + dishes_id: 02543 (number, required) - different dishes' id should be different.
+        + vendorname: `蘭媽` (string, required) -  the name of the restrunt which user chose.
```
Comment (barrychung1112) [inline client/api_spec/dishes_control_api.md:14]: restaurant! not 'restrunt'.

File: client/api_spec/dishes_control_api.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        + dishes_name: `無骨雞排` (string, required) - the name of the meal which user chose.
```
Comment (barrychung1112) [inline client/api_spec/dishes_control_api.md:15]: Why `dishes_name` in type string? Only one dish for an order?

Comment (Brianlih) [reply client/api_spec/dishes_control_api.md:15]: I have fixed it.

File: client/api_spec/dishes_control_api.md (hunk 3 above)
Comment (Brianlih) [reply client/api_spec/dishes_control_api.md:14]: I have fixed it.

File: client/api_spec/dishes_control_api.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+# Group Dishes_Control API
+
+## Admin Dishes_Control [/api/dishes_control]
+
+### Create Order to complete order list [POST /api/dishes_control/complete_order_list]
+
++ Request (application/json)
+    + Attributes
+        + dishes_id: 02543 (number, required) - different dishes' id should be different.
```
Comment (barrychung1112) [inline client/api_spec/dishes_control_api.md:13]: Do you mean dish's?

**Pull request merged**